modules/ROOT/pages/configure-acm.adoc

```diff
@@ -490,21 +484,18 @@ If you don't configure an approval process, self-registered users will become me
 ... *Value*: `False`
 .. Select *Next*.
 
-.. In *Step 3. Specify Approver Field and Record Editability Properties*:
-... In the *Record Editability Properties* section select *Administrators ONLY can edit records during the approval process*.
+.. In *Step 3. Specify Approver Field and Record Editability Properties*, in the *Record Editability Properties* section select *Administrators ONLY can edit records during the approval process*.
 .. Select *Next*.
 
-.. In *Step 4. Select Notification Templates*:
-... Select *Registration Request* in the *Approval Assignment Email Template* section, under the *ACM* category in the look-up dialog.
+.. In *Step 4. Select Notification Templates*, select *Registration Request* in the *Approval Assignment Email Template* section, under the *ACM* category in the look-up dialog.
```

Comment:
We need to find a way to reword/re-present these circuitous verbal navigations....

Reply:
Agreed.